### PR TITLE
Implement mechanisms to force an agent reconnection with the manager - Documentation changes

### PR DESCRIPTION
**Pull request by DProvinciani**
## Description

This pull request includes the description of the new `force_reconnect_interval` setting added as part of the epic https://github.com/wazuh/wazuh/issues/7886.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns.

### CODE REVIEW

File: source/user-manual/reference/ossec-conf/client.rst [hunk 1]
```diff
@@ -233,6 +234,19 @@ For example, a ``notify_time`` setting of 60 combined with a time-reconnect of 3
 .. warning::
 	Notice that the ``notify_time`` value uses an underscore while the ``time-reconnect`` value uses a dash.  This is an unfortunate legacy naming inconsistency that is easy to mix up.
 
+  .. _force_reconnect_interval:
+
+force_reconnect_interval
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Specifies the time after which the agent is forced to be reconnected to the manager. The reconnection is forced to be done independently if a checkin attempt failed to get a response from the manager.
```
Comment (palaciosjeremias) [inline source/user-manual/reference/ossec-conf/client.rst:242]: Maybe we can express this like:
"The reconnection is forced to be done even if the agent is having a successful communication with the manager from the manager."

File: source/user-manual/reference/ossec-conf/client.rst [hunk 2]
```diff
@@ -233,6 +234,19 @@ For example, a ``notify_time`` setting of 60 combined with a time-reconnect of 3
...
+
++--------------------+------------------------------------------------------------------------------------------------------------------------------------+
+| **Default value**  | 0s (disabled)                                                                                                                      |
++--------------------+------------------------------------------------------------------------------------------------------------------------------------+
+| **Allowed values** | A positive number that should end with a character indicating a time unit, such as: s (seconds), m (minutes), h (hours), d (days). |
```
Comment (palaciosjeremias) [inline source/user-manual/reference/ossec-conf/client.rst:247]: We should add **w (weeks)** option here too.

File: source/user-manual/reference/ossec-conf/client.rst (hunk 1 above)
Comment (DProvinciani) [reply source/user-manual/reference/ossec-conf/client.rst:242]: Fixed

File: source/user-manual/reference/ossec-conf/client.rst (hunk 2 above)
Comment (DProvinciani) [reply source/user-manual/reference/ossec-conf/client.rst:247]: Fixed.

**Review by palaciosjeremias — APPROVED** [review]
LGTM!

**Review by davidfmiranda — APPROVED** [review]
LGTM

File: source/user-manual/reference/ossec-conf/client.rst [hunk 3]
```diff
@@ -233,6 +234,19 @@ For example, a ``notify_time`` setting of 60 combined with a time-reconnect of 3
 .. warning::
 	Notice that the ``notify_time`` value uses an underscore while the ``time-reconnect`` value uses a dash.  This is an unfortunate legacy naming inconsistency that is easy to mix up.
 
+  .. _force_reconnect_interval:
```
Comment (vikman90) [inline source/user-manual/reference/ossec-conf/client.rst:237]: We need to add information about the version that introduces this option (4.3.0).

**Review by vikman90 — CHANGES_REQUESTED** [review]

File: source/user-manual/reference/ossec-conf/client.rst (hunk 3 above)
Comment (DProvinciani) [reply source/user-manual/reference/ossec-conf/client.rst:237]: Fixed

**Pull request merged**